stock_picking_comment_template/models/stock_picking.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 C2i Change 2 improve - Eduardo Magdalena <emagdalena@c2i.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    """Add text comment"""
+
+    _inherit = "stock.picking"
+
+    comment_template1_id = fields.Many2one('base.comment.template',
+                                           string='Top Comment Template')
+    comment_template2_id = fields.Many2one('base.comment.template',
+                                           string='Bottom Comment Template')
+    note1 = fields.Html('Top Comment')
+    note2 = fields.Html('Bottom Comment')
+
+    @api.onchange('comment_template1_id')
+    def _set_note1(self):

```

Comment:
@emagdalenaC2i Functionally, is it good to erase a  note field that could have been modified by user (he maybe doesn't want to have his modifications lost)?